Tests/IntegrationTests/SharedNodeTypesPackage/NodeTypes/Content/InlineHeadline.yaml

```diff
@@ -21,7 +21,7 @@
         inlineEditable: true
         inline:
           editorOptions:
-            autoparagraph: false
+            isInline: true
```

Comment:
- inlineMode?
- inlineContentMode?
- block = false

or decide in fusion?

```
Neos.Fusion:Editable {
   block = false
}
```

Reply:
We decided to use the the same naming as in fusion: `block` false.

And its allowed to either specify it in the nodeType or via the rendering.

Following logic would apply:

```js
    const isInlineMode = editorOptions?.block === false ||
         propertyDomNode.tagName === 'SPAN' ||
         propertyDomNode.tagName === 'H1' ||
         propertyDomNode.tagName === 'H2' ||
         propertyDomNode.tagName === 'H3' ||
         propertyDomNode.tagName === 'H4' ||
         propertyDomNode.tagName === 'H5' ||
         propertyDomNode.tagName === 'H6' ||
         propertyDomNode.tagName === 'P';
```


happy cases:

`editorOptions: { } | { block: true }` + `Neos.Fusion:Editable { } | { block: true }` -> `<div>{isInlineMode: false}</div>` 
`editorOptions: { }` + `Neos.Fusion:Editable { block: false }` -> `<span>{isInlineMode: true}</span>` 
`editorOptions: { block: false }` + `Neos.Fusion:Editable { block: false }` -> `<span>{isInlineMode: true}</span>` 

but this would also lead to cases like this:

`editorOptions: { block: false }` + `Neos.Fusion:Editable { } | { block: true }` -> `<div>{isInlineMode: true}</div>` 
`editorOptions: { block: true }` + `Neos.Fusion:Editable { block: false }` -> `<span>{isInlineMode: false}</span>` 


So the the notation of `block: false` will always overrule the one from the `Neos.Fusion:Editable`. But the rendering will still be affected via `Neos.Fusion:Editable` 
(i hope i have no logic error in my equations ^^)